### DISCUSSION TITLE
Black Litterman - Confidence Intervals approach (Idzorek's Method)

### DISCUSSION THREAD

**Discussion by FeCol150** [dcajasn/Riskfolio-Lib#165] [Q&A]
Hi, 

I am working on a BL Idzorek's Method which uses analysts' views confidence interval. I have not been able to upload the confidence interval trough ConstraintsFunctions.assets_views(views, asset_classes). On the other hand, It is not clear for me yet if riskfolio-lib has the function available to get an optimization with confidence intervals.

**Comment by dcajasn** [comment]
Hi,

It's not implemented. There is no plan to implement Idzorek's method in short or near term.

Best,
Dany

**Reply by FeCol150** [reply]
Thank you for the answer. Btw. Also, thank you for you development on the library.
